crates/emitter/src/aligner.rs

```diff
@@ -203,6 +203,30 @@ impl Aligner {
             .implicit_parameter_types
             .contains(&BuiltinType::Type)
     }
+
+    fn identifier_with_prefix_suffix(
```

Comment:
How about making this implementation the same as `identifier_with_prefix_suffix` in emitter?

Reply:
I've resolved it.